ark/type/__tests__/keywords/string.test.ts

```diff
@@ -20,6 +20,14 @@ contextualize(() => {
 		)
 	})
 
+	it("hex", () => {
+		const hex = type("string.hex")
+		attest(hex("1fA3").toString()).equals("1fA3")
+		attest(hex("0x1A3").toString()).equals('must be hex (was "0x1A3")')
```

Comment:
Let's make the description a little more detailed e.g. "only hex characters" or similar